api.go

```diff
@@ -0,0 +1,80 @@
+package main
+
+import (
+	"fmt"
+	"os/exec"
+)
+
+func btcctlCommand(args []string) {
+	argsLen := len(args)
+	if argsLen == 0 {
+		fmt.Println("No commands were provided")
+		return
+	}
+
+	btcctlConfPath := "/Users/draku/go/src/orcacointest/cmd/btcctl/sample-btcctl.conf"
+	argIdx := 0
+	for argIdx < argsLen {
+		arg := args[argIdx]
+		switch arg {
+		case "getbalance":
+			cmd := exec.Command("btcctl", "--configfile="+btcctlConfPath, "getbalance", "--notls")
+			output, err := cmd.Output()
+			if err != nil {
+				fmt.Println("Error when running 'getbalance':", err)
+				return
+			}
+
+			fmt.Println(string(output))

```

Comment:
The steps to run each command have a lot of similar code. You can create a function and move the common code there. Something like - 

```
func executeCommand(command string, args ...string) error {
	cmd := exec.Command("btcctl", "--configfile="+btcctlConfPath, command, args, "--notls")
	output, err := cmd.Output()
	if err != nil {
		fmt.Println("error running '%s': %w", command, err)
		return
	}
	fmt.Println(string(output))
}

executeCommand(generateCommand, args[argIdx+1], args[argIdx+2])
```

Reply:
Fixed